Assets/Scripts/Characters/Human/Hero.cs

```diff
@@ -747,6 +747,19 @@ public void checkTitan()
         this.myTitans = list2;
     }
 
+    private void OnCollisionEnter(Collision collision)
+    {
+        
+        float force = collision.impulse.magnitude / Time.fixedDeltaTime;
```

Comment:
Why should the force be divided by Time.fixedDeltaTime? This value is always 0.02.

Reply:
This was included in the paper linked in the original thread (https://www.malgol.com/how-to-get-the-impact-force-of-a-collision-in-unity/). I dont understand the physics described, but if needed, i'll investigate some more.